common/set-login-build-task.licenseheader

```diff
@@ -1,14 +0,0 @@
-﻿extensions: .ts
-// Copyright %CreationYear% Google Inc. All Rights Reserved
```

Comment:
Why are all of these being removed?

Reply:
They are artifacts of a particular VS Extension. They should not have been in the repository in the first place.